frontend/wasm/src/api.rs

```diff
@@ -31,8 +31,11 @@ pub struct Editor {
 impl Editor {
 	#[wasm_bindgen(constructor)]
 	pub fn new(handle_response: js_sys::Function) -> Editor {
+		let editor_id = generate_uuid();
+		let editor = editor::Editor::new();
+		EDITOR_INSTANCES.with(|instances| instances.borrow_mut().insert(editor_id, editor));
 		Editor {
-			editor: UnsafeCell::new(editor::Editor::new()),
+			editor_id,
 			instance_received_crashed: Cell::new(false),
 			handle_response,
 		}
```

Comment:
Could we make this naming a little less confusing? Having two `Editor`s is a bit confusing, had to read the function twice to understand what was going on.